contents/docs/experiments/manual.mdx

```diff
@@ -8,136 +8,303 @@ availability:
+<MultiLanguage>
 
-PostHog computes this significance for you automatically - we will let you know if your experiment has reached significant results or not. Once your experiment reaches significant results, it's safe to use those results to reach a conclusion and terminate the experiment. You can read more about how we do this in our 'Advanced' section below.
+```node
+client.capture({
+    distinctId: 'distinct_id',
+    event: 'event_name_of_your_goal_metric',
+    properties: {
+        '$feature/experiment_feature_flag_name': 'variant_name'
+    },
+})
+```
 
-### Experimentation vs. Feature Flags
+```python
+posthog.capture(
+    'distinct_id',
+    'event_name_of_your_goal_metric',
+    {
+        '$feature/experiment_feature_flag_name': 'variant_name'
+    }
+)
+```
 
-Experimentation and Feature Flags are different features because they serve different purposes. **Experimentation** is used to test changes to your product where you want to maximize a specific metric (eg. testing a new flow to maximize conversion rate). **Feature Flags** are used for phased roll-outs (e.g. a new experimental feature you want to make sure works correctly) or as a way to control feature access. Read our [Feature Flags user guide](/docs/user-guides/feature-flags) for more information on this.
+```php
+PostHog::capture(array(
+  'distinctId' => 'distinct_id',
+  'event' => 'event_name_of_your_goal_metric',
+  'properties' => array(
+    '$feature/experiment_feature_flag_name'  = 'variant_name'
+  )
+));
+```
 
-Experimentation uses multivariate Feature Flags under the hood to handle user allocation to an experiment. You'll notice this when checking if a user is assigned to an experiment.
+```ruby
+posthog.capture({
+    distinct_id: 'distinct_id',
+    event: 'event_name_of_your_goal_metric',
+    properties: {
+        '$feature/experiment_feature_flag_name': 'variant_name',
+    }
+})
+```
 
-While Feature Flags can be boolean or multivariate, Experimentation always uses a multivariate approach.
+```go
+client.Enqueue(posthog.Capture{
+  DistinctId: "distinct_id",
+  Event:      "event_name_of_your_goal_metric",
+  Properties: posthog.NewProperties().
+    Set("$feature/experiment_feature_flag_name", "variant_name"),
+})
+```
+
+```java
+posthog.capture(
+  "distinct_id",
+  "event_name_of_your_goal_metric",
+  new HashMap<String, Object>() {
+    {
+      put("$feature/experiment_feature_flag_name", "variant_name");
+    }
+  }
+);
+```
+
+</MultiLanguage>
+
+## Launching your experiment
+
+Once you've written your code, it's a good idea to test that each variant behaves as you'd expect. If you find out your implementation had a bug after you've launched the experiment, you lose days of effort as the experiment results can no longer be trusted.
```

Comment:
How do you test?

Reply:
variant overrides - is the way to go, we should promote this over the toolbar, which can get unreliable.

Reply:
I added the variant override code snippet to the JS and react code, but it's not available in the other SDKs

Reply:
Thanks! I'll include this